assets/js/common/SuseManagerConfig/SuseManagerConfig.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { defaultTo, noop } from 'lodash';
+
+import Button from '@common/Button';
+
+import CertificateUploadDate from './CertifiedUploadDate';
+
+const renderPassword = (username, certUploadDate) =>
+  username && certUploadDate ? '•••••' : '.....';
```

Comment:
Looks great :rocket: I'm just curious why here we use `'.....'` for the empty state, instead of `'-'` like in `CertificateUploadDate`?

Reply:
I just copied the mockup without asking :grimacing: @jagabomb what do you think?

Reply:
I like it the way it is for now. In the design, I just put the ellipsis(...) for username as a way to show there needs to be a value set by the user. Unlike the certificate field which can stay empty and hence the use of the dash (–). I also considered writing Not Applicable/
Available (N/A) for the certificate field but I thought it might imply too much or lead to confusion.

Reply:
Thanks both for the suggestions!